types/crypto-js/crypto-js-tests.ts

```diff
@@ -11,6 +11,10 @@ wordArray = CryptoJS.SHA1('some message', 'some key', { any: true });
 wordArray = CryptoJS.format.OpenSSL('some message');
 wordArray = CryptoJS.format.OpenSSL('some message', 'some key');
 
+var FR = new FileReader();
+FR.onloadend = () => {
+    var hash = CryptoJS.SHA1(CryptoJS.lib.WordArray.create(FR.result)).toString()
```

Comment:
Please try to keep same indention in file. Here are used tabs instead of spaces.